### ISSUE TITLE
buttons unlabeled when using screen reader

### ISSUE THREAD

**Issue by crentava** [authpass/authpass#214]
I recently started using AuthPass and it is one of the best password managers I’ve used. Some of the buttons however are unlabeled and for me (as a blind user) I don’t know what purpose they serve. Could these buttons please be labeled? to make it more convenient I will give a list of these buttons

 1. The search button
 2. buttons after group names in customize menu
 3. in the manage groups section, the button before the “groups” heading as well as the buttons after the name of the groups you have
 4. in the “preferences” section, The three buttons after “visual density” and “text scale factor”
 5. The button to add a new password
thank you so much and if I missed any unlabeled buttons I will either use this issue or open another one

**Comment by hpoul** [comment]
Thanks for the heads up, I've never actually dealt with screen readers. I will look into it.

**Comment by hpoul** [comment]
I have added a few more semantic labels. Which platforms are you using, maybe you can help me test it? most dev versions should be available soon (see https://authpass.app/docs/help-us-beta-testing/ )

**Comment by crentava** [comment]
@hpoul I am an iOS user. I do have an android laying around but I’d have to find the charger for it. If you’d like I can open other issues as well about accessibility. I think more people should be using this and accessibility improvements may help

**Comment by hpoul** [comment]
You can also join the testflight beta for iOS, it should be getting there soon'ish.
Yeah sure, feel free to create more issues.. maybe group them by topic or screen if there are many.

**Comment by crentava** [comment]
just installed the beta. Thank you so much, I’ll be waiting to see the improvements

**Comment by hpoul** [comment]
fyi, i've had a few problems with the CI upload to testflight, but it should be online now.. maybe you can give it a try

**Comment by crentava** [comment]
i don’t see an update in testFlight

**Comment by hpoul** [comment]
sorry, used the wrong tester group, should be available now

**Comment by crentava** [comment]
most of the buttons are labeled now, thank you. If you don’t mind I opened up another issue yesterday about the keyboard. I’m gonna close this one and hopefully the other ones will be able to be fixed so that things will be easier. Thanks again
